packages/backend/src/graphql/resolvers/liquidity.ts

```diff
@@ -100,7 +100,8 @@ export const depositPeerLiquidity: MutationResolvers<ApolloContext>['depositPeer
     const peerOrError = await peerService.depositLiquidity({
       transferId: args.input.id,
       peerId: args.input.peerId,
-      amount: args.input.amount
+      amount: args.input.amount,
+      tenantId: ctx.tenant.id
```

Comment:
```suggestion
      tenantId: ctx.isOperator ? undefined : ctx.tenant.id
```
so the operator can create liquidity deposits for any peer